### PR TITLE
feat: Adjust checkout button

### PR DESCRIPTION
**Pull request by ArthurTriis1**
Signed-off-by: Arthur Andrade <arthurfelandrade@gmail.com>

## What's the purpose of this pull request?
Fix OrderSummary stats in CarSidebar, now when not exist products in cart the OrderSummary not render

### CODE REVIEW

**Review by saranicoly — APPROVED** [review]
LGTM! 👏

**Review by danzanzini — APPROVED** [review]

**Review by israelswf — APPROVED** [review]
✨

File: src/components/cart/CartSidebar/CartSidebar.tsx [hunk 1]
```diff
@@ -30,24 +30,28 @@ function CartSidebar() {
...
-            variant="primary"
-            icon={<ArrowRightIcon size={18} />}
-            iconPosition="right"
-            {...btnProps}
-          >
-            {isValidating ? 'loading...' : 'Checkout'}
-          </Button>
-        }
-      />
+      {!!totalItems && (
```
Comment (lucasfp13) [inline src/components/cart/CartSidebar/CartSidebar.tsx:33]:
```suggestion
      {totalItems && (
```

**Review by lucasfp13 — APPROVED** [review]
LGTM! 💯

**Pull request merged**